docs/docs/guides/config/server_admin.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Server Instance superuser Password Reset
```

Comment:
update title? is `Controller Admin Settings` good for the title?

Reply:
done